static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -149,78 +123,99 @@ const getRules = (): Object => {
         minAbove: 300,
-        minBelow: 300,
+        minBelow: 0,
         selectors: {
-            ' .ad-slot': adSlotClassSelectorSizes,
-            ' .ad-slot--inline1': {
+            ' > .merch': {
                 minAbove: 0,
-                minBelow: blockadeMinAbove,
+                minBelow: 0,
+            },
+            ' > header': {
+                minAbove: isBreakpoint({
+                    max: 'tablet',
+                })
+                    ? 300
+                    : 700,
+                minBelow: 0,
+            },
+            ' > h2': {
+                minAbove: 100,
+                minBelow: 250,
+            },
+            ' .ad-slot': adSlotClassSelectorSizes,
+            ' > :not(p):not(h2):not(.ad-slot)': {
+                minAbove: 200,
+                minBelow: 400,
             },
         },
-        filter,
     };
 
-    return shouldUseVariantRules && bodyAds > 0 ? variantRules : defaultRules;
-};
-
-const addInlineAds = (): Promise<number> =>
-    addArticleAds(getRules(), 1).then(() => addArticleAds(getRules()));
-
-const getInlineMerchRules = (): Object => {
-    const inlineMerchRules: Object = getRules();
-    inlineMerchRules.minAbove = 300;
-    inlineMerchRules.selectors[' > h2'].minAbove = 100;
-    inlineMerchRules.selectors[
-        ' > :not(p):not(h2):not(.ad-slot)'
-    ].minAbove = 200;
-    return inlineMerchRules;
-};
-
-const addInlineMerchAd = (): Promise<any> =>
-    spaceFiller.fillSpace(
-        getInlineMerchRules(),
-        paras => insertAdAtPara(paras[0], 'im', 'im').then(() => 1),
+    return spaceFiller.fillSpace(
+        rules,
+        paras => insertAdAtPara(paras[0], 'im', 'im').then(() => true),
         {
             waitForImages: true,
             waitForLinks: true,
             waitForInteractives: true,
         }
     );
+};
 
-const waitForMerch = (countAdded: number): Promise<void> =>
-    countAdded === 1 ? trackAdRender('dfp-ad--im') : Promise.resolve();
-
-export const init = (start: () => void, stop: () => void): Promise<boolean> => {
+export const init = (start: () => void, stop: () => void): Promise<any> => {
     start();
     if (!commercialFeatures.articleBodyAdverts) {
         stop();
-        return Promise.resolve(false);
-    }
-
-    bodyAds = 0;
-
-    if (config.page.hasInlineMerchandise) {
-        const im = addInlineMerchAd();
-        // Whether an inline merch has been inserted or not,
-        // we still want to try to insert inline MPUs. But
-        // we must wait for DFP to return, since if the merch
-        // component is empty, it might completely change the
-        // positions where we insert those MPUs.
-        im
-            .then(waitForMerch)
-            .then(addInlineAds)
-            .then(stop);
-
-        return im;
+        return Promise.resolve();
     }
 
-    addInlineAds().then(stop);
-    return Promise.resolve(true);
+    const im = config.page.hasInlineMerchandise
```

Comment:
Could use `config.get(...)` for safer accessing? However, appreciate that this is code from before

Reply:
Yep, good shout - I can update that 👍 

Need to fix the tests too 😩